config/locales/en.yml

```diff
@@ -103,6 +103,13 @@ en:
         link_text: Check what you need to do
         title: Brexit
       world_locations: World locations
+    save_this_page:
+      page_not_saved_heading: Save this page so you can come back to it later
+      page_was_saved_heading: You’ve saved this page
+      add_page_button: Add to your saved pages
+      remove_page_button: Remove from your saved pages
+      see_saved_pages_signed_in: <a href="/asdf-am-signedin" class="govuk-link">See your saved pages</a>
+      see_saved_pages_signed_out: <a href="/asdf-am-signedout" class="govuk-link">Sign in</a> to see your saved pages.
```

Comment:
What are `/asdf-am-signedin` and `/asdf-am-signedout`?

Reply:
@danacotoran can we make it clearer that these are just placeholders using something like
`/placeholder-signedin-endpoint`
and
 `/placeholder-signedout-endpoint`
 ?